### DISCUSSION TITLE
How do the Slack integration works?

### DISCUSSION THREAD

**Discussion by phyx3r** [botfront/botfront#928] [Q&A]
Hi there, I've seen a Links for Handoff setup section on the admin panel.
In this section there is an input field for Slack with no hints or explanation about how to use it.

How it supposed to work and what should I input in that field?

I'm very interested in handoff feature.
Thank you very much!

**Comment by phyx3r** [comment]
Any hint on that? Best

**Comment by thomas779** [comment]
Check out this thread on the rasa forums: https://forum.rasa.com/t/handing-off-to-human-on-slack/26203 
This thread too: https://forum.rasa.com/t/forwarding-messages-to-slack-and-vice-versa-for-human-hand-off/28447
